cogs/Music.py

```diff
@@ -0,0 +1,360 @@
+    
+    def npEmbed(self, ctx, player):
+        sean = time.time() - player.timer
+        elapsed = time.strftime("%M:%S", time.gmtime(sean))
+        length = time.strftime("%M:%S", time.gmtime(player.audioSource.data['duration']))
+        timerStr = self.timerString(ctx, sean, player.audioSource.data['duration'])
+        
+        embed = discord.Embed(color=0xBEDCF6)
+        embed.set_thumbnail(url=f"{player.audioSource.data['thumbnail']}")
+        embed.add_field(name="Now Playing", value=f"""
+                        [{player.audioSource.title}]({player.audioSource.data['webpage_url']})
+                        From {player.audioSource.data['uploader']}
+                        
+                        `{timerStr}`
+                        Duration: `{elapsed} / {length}`""", 
+                        inline=False)
+        embed.set_footer(text=f"Requested by {player.requester}")
+        return embed
+        
+    def queueEmbed(self, ctx, player):
+        length = time.strftime("%M:%S", time.gmtime(player.audioSource.data['duration']))
+        embed = discord.Embed(color=0xBEDCF6)
+        embed.set_thumbnail(url=f"{player.audioSource.data['thumbnail']}")
+        embed.add_field(name="Added to Queue", value=f"""
+                        [{player.audioSource.title}]({player.audioSource.data['webpage_url']})
+                        From {player.audioSource.data['uploader']}
+                        
+                        Duration: `{length}`""", 
+                        inline=False)
+        embed.set_footer(text=f"Requested by {player.requester}")
+        return embed
+    
+    def write(self, ctx, start, player):
+        embed = discord.Embed(color=0xBEDCF6)
+        embed.set_thumbnail(url=f"{player[start].audioSource.data['thumbnail']}")
+        
+        iteration = 0
+        duration = 0
+        while start < len(player) and iteration < 5: #Will loop until 5 entries are processed or there's nothing left in the queue
+            length = time.strftime("%M:%S", time.gmtime(player[start].audioSource.data['duration']))
+            duration += player[start].audioSource.data['duration']
+            embed.add_field(
+                name = f"{player[start].audioSource.title}",
+                value = f"By `{player[start].audioSource.data['uploader']}`\nLength: `{length}`\nAdded by `{player[start].requester}`",
+                inline = False
+                )
+            iteration += 1
+            start +=1 
+        if duration < 3600:
+            duration = time.strftime("%M:%S", time.gmtime(duration))
+        else:
+            duration = time.strftime("%H:%M:%S", time.gmtime(duration))
+        if len(player) > 1:
+            embed.set_footer(text=f"Total length of this queue is {len(player)} songs, {duration}")
+        else:
+            embed.set_footer(text=f"Total length of this queue is {len(player)} song, {duration}")
+        return embed
+    
+#    def listQueueEmbed(self, ctx, pQueue):

```

Comment:
I don't understand the purpose of these comments

Reply:
Keep it there for the moment. That's the old queue command